service/debugger/debugger.go

```diff
@@ -114,6 +114,9 @@ func (d *Debugger) Restart() error {
 		return fmt.Errorf("could not launch process: %s", err)
 	}
 	for _, oldBp := range d.breakpoints() {
+		if oldBp.ID < 0 {
```

Comment:
Because recreating the `Process` already sets the breakpoint, setting it here would cause an error, which would have to be ignored.